go-controller/pkg/ovn/egressgw.go

```diff
@@ -494,33 +510,98 @@ func (oc *Controller) addHybridRoutePolicyForPod(podIP net.IP, node string) erro
 // by ecmp routes
 func (oc *Controller) delHybridRoutePolicyForPod(podIP net.IP, node string) error {
 	if config.Gateway.Mode == config.GatewayModeLocal {
-		// delete allow policy to bypass lr-policy in GR
+		// Delete podIP from the node's address_set.
+		as, err := oc.addressSetFactory.EnsureAddressSet("hybrid-route-pods-" + node)
```

Comment:
nit: should move this prefix into `const.go`. Will wait and see if there are other comments in this PR to fix and do them together, else will do a follow-up